feature-libs/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.html

```diff
@@ -63,66 +63,89 @@
     </div>
 
     <!-- mobile tabs for column set only -->
-
     <div *ngIf="locations?.stores" class="cx-columns-mobile">
-      <ul ngbNav #nav="ngbNav">
-        <li ngbNavItem>
-          <a ngbNavLink>
-            {{ 'storeFinder.listView' | cxTranslate }}
-          </a>
-          <ng-template ngbNavContent>
-            <div class="cx-address-col">
-              <div class="cx-store-details" *ngIf="isDetailsModeVisible">
-                <cx-store-finder-store-description
-                  [location]="storeDetails"
-                  [disableMap]="true"
-                ></cx-store-finder-store-description>
+      <ul role="tablist" class="nav">
+        <li class="nav-item" *ngFor="let mode of displayModes | keyvalue">
```

Comment:
We should add the `cx-` prefix to the nav-item class

Reply:
Same case as above.